improver_tests/generate_ancillaries/test_DistanceTo.py

```diff
@@ -397,27 +399,62 @@ def test_distance_to_with_polygon_geometry(
 
 
 @pytest.mark.parametrize(
-    "geometry_type,expected_distance",
+    "geometry_type,distance_type,if_parallel,expected_distance",
     [
-        ("point", [0, 500, 707, 707]),
-        ("line", [0, 0, 500, 500]),
-        ("polygon", [0, 0, 0, 500]),
+        ("point", "nearest", False, [0, 500, 707, 707]),
+        ("line", "nearest", False, [0, 0, 500, 500]),
+        ("polygon", "nearest", False, [0, 0, 0, 500]),
+        ("point", "nearest", True, [0, 500, 707, 707]),
+        ("line", "nearest", True, [0, 0, 500, 500]),
+        ("polygon", "nearest", True, [0, 0, 0, 500]),
+        ("point", "furthest", True, [1414, 1118, 707, 1581]),
+        ("line", "furthest", True, [0, 0, 500, 500]),
+        ("polygon", "furthest", True, [0, 0, 0, 500]),
     ],
 )
-@pytest.mark.parametrize("geometry_crs", ("laea", "latlon"))
+@pytest.mark.parametrize("geometry_crs", ("laea",))
 def test_distance_to_with_multiple_sites(
     multiple_site_cube,
     geometry_type,
     geometry_crs,
     expected_distance,
+    distance_type,
+    if_parallel,
     request,
 ):
     """Test the DistanceTo plugin works when provided a site cube with multiple sites and
     different types of geometry"""
 
     geometry = request.getfixturevalue(f"geometry_{geometry_type}_{geometry_crs}")
 
-    output_cube = DistanceTo(3035)(multiple_site_cube, geometry)
+    n_jobs = 1
+    if os.cpu_count() > 1:
+        n_jobs = 2
+
+    plugin = DistanceTo(3035, parallel=if_parallel, n_parallel_jobs=n_jobs)
+    # Modify the plugin to use a custom distance calculation method when
+    # parallel processing is requested with the 'furthest' processing mode.
+    # This demonstrates that the plugin must be using the parallel processing pathway.
+    if if_parallel and distance_type == "furthest":
```

Comment:
I don't like this invention of an unsupported "furthest" option, and rewriting of the code here, just to demonstrate that the code has taken a particular path. Why would we doubt the if statement doing what we expect? 

If you want to check that the parallel argument is set correctly you could inspect the plugin instance, but beyond that if the code doesn't fall over and we get a sensible result, surely it must have gone through the `parallel` wrapper if `self.parallel` is True.

Reply:
I've removed this invention, which simplifies the tests.